src/lib/config/trace.go

```diff
@@ -0,0 +1,37 @@
+package config
+
+import (
+	"context"
+
+	"github.com/goharbor/harbor/src/common"
+	commonhttp "github.com/goharbor/harbor/src/common/http"
+	"github.com/goharbor/harbor/src/lib/log"
+	tracelib "github.com/goharbor/harbor/src/lib/trace"
+)
+
+func InitTraceConfig(ctx context.Context) {
+	cfgMgr, err := GetManager(common.InMemoryCfgManager)
```

Comment:
Is it right to use the memory rather than restcfgManager? @stonezdj correct me.

Reply:
rest Manager fetch data through the core component, but some data are special for jobservice like service name, so it need to parse its own env

Reply:
But in the case of tracing, the env var like `TRACE_SERVICE_NAME` has different values for harbor-core and harbor-job service. But if we use the `RestConfigManager` the value will be the same as harbor-core which is not our intention. As a result, either way, we need an `InMemoryCfgManager`.